frontend/src/metabase/core/components/Toggle/Toggle.styled.tsx

```diff
@@ -2,56 +2,60 @@ import styled from "styled-components";
 import { color } from "metabase/lib/colors";
 
 export interface ToggleRootProps {
-  isSmall?: boolean;
-  isSelected?: boolean;
+  checked?: boolean;
+  small?: boolean;
   currentColor?: string;
 }
 
-const getLeft = ({ isSmall, isSelected }: ToggleRootProps): string => {
-  if (!isSelected) {
+const getTranslateX = ({ checked, small }: ToggleRootProps): string => {
+  if (!checked) {
     return "1px";
-  } else if (!isSmall) {
+  } else if (!small) {
     return "25px";
   } else {
     return "12px";
   }
 };
 
 const getBackgroundColor = ({
-  isSelected,
+  checked,
   currentColor,
 }: ToggleRootProps): string => {
-  if (isSelected) {
+  if (checked) {
     return currentColor ?? color("brand");
   } else {
     return color("bg-medium");
   }
 };
 
-export const ToggleRoot = styled.a<ToggleRootProps>`
+export const ToggleRoot = styled.input<ToggleRootProps>`
+  appearance: none;
   position: relative;
   display: inline-block;
   color: ${props => props.currentColor ?? color("brand")};
-  box-sizing: border-box;
```

Comment:
Curious about this removal. Is the idea that we already define this at `*` in our global CSS, or was there another reason?

Reply:
Yes, it is defined for `input`s.